### PR TITLE
Description of routing parameters + example

### PR DESCRIPTION
**Pull request by straakaa**

### CODE REVIEW

File: bobril-page/pages/moreTutorials/content.tsx [hunk 1]
```diff
@@ -305,10 +305,18 @@ describe("Todo store", () => {
...
 <p>{`Bobril has few methods for defining the application route tree:`}</p>
 <ul>
-<li><code>b.route</code>{` - defines a route url, name, handler and a list of sub-routes`}</li>
+<li><code>b.route</code>{` - defines route parameters and a list of sub-routes`}</li>
 <li><code>b.routes</code>{` - registers routes to the application and calls `}<code>b.init</code></li>
 <li><code>b.routeDefault</code>{` - defines the default route if no sub-route is specified in the current url`}</li>
 </ul>
+<p>{`Route parameters:`}</p>
+<ul>
+<li><code>url</code>{` - Define the name of the route. The name is used as an identifier during redirection. Name cannot contain “:” or “/“`}</li>
```
Comment (Bobris) [inline bobril-page/pages/moreTutorials/content.tsx:314]: swapped url and name?

File: md/more-tutorials/04-routing.md [hunk 2]
```diff
@@ -4,10 +4,18 @@
...
 Bobril has few methods for defining the application route tree:
 
-- `b.route` - defines a route url, name, handler and a list of sub-routes
+- `b.route` - defines route parameters and a list of sub-routes
 - `b.routes` - registers routes to the application and calls `b.init`
 - `b.routeDefault` - defines the default route if no sub-route is specified in the current url
 
+Route parameters:
+
+- `url` - Define the name of the route. The name is used as an identifier during redirection. Name cannot contain ":" or "/"
```
Comment (Bobris) [inline md/more-tutorials/04-routing.md:13]: Actually url and name swapped here

File: bobril-page/pages/moreTutorials/content.tsx (hunk 1 above)
Comment (straakaa) [reply bobril-page/pages/moreTutorials/content.tsx]: yes, I'll fix it

**Comment by Bobris** [comment]
Git history will be a mess. 😜

**Comment by straakaa** [comment]
> Git history will be a mess. 😜

Sorry :( should I delete it and make fork again ?

**Comment by Bobris** [comment]
It looks good after merge.

**Pull request merged**